criblio/appscope
# [#1260] Connection status information

QA Instructions:
- Start scoping the process.
- Inspect the scoped process
```
scope inspect <SCOPED_PROCESS_PID>
```
Example interface section:
```
   "interfaces": [
      {
         "name": "log",
         "connected": true,
         "config": "file:///tmp/scope.log"
      },
      {
         "name": "cribl",
         "connected": false,
         "config": "edge",
         "attempts": 4
      }
   ]
```

The output should reflect the used configuration and the current status of the connection.
Please use different configurations to verify that all cases are covered.

Ref: #1260
